lms/static/js/groups/views/certificate_whitelist_editor.js

```diff
@@ -0,0 +1,86 @@
+// Backbone Application View: CertificateWhitelist View
```

Comment:
Put all of these certificate exception related models, views, collections in `lms/static/js/certificates`.